sdk/digitaltwins/digital-twins-core/src/digitalTwinsClient.ts

```diff
@@ -859,20 +859,20 @@ export class DigitalTwinsClient {
 
   /**
    * Decommission a model using a json patch.
-   * When a model is decomissioned, new digital twins will no longer be able to be
+   * When a model is decommissioned, new digital twins will no longer be able to be
    * defined by this model. However, existing digital twins may continue to use this model.
-   * Once a model is decomissioned, it may not be recommissioned.
+   * Once a model is decommissioned, it may not be recommissioned.
    *
    * @param modelId - The Id of the model to decommission.
    * property can be replaced.
    * @param options - The operation options
    * @returns The http response.
    *
    */
-  public decomissionModel(modelId: string, options: OperationOptions = {}): Promise<RestResponse> {
+  public decommissionModel(modelId: string, options: OperationOptions = {}): Promise<RestResponse> {
     const jsonPatch = [{ op: "replace", path: "/decommissioned", value: true }];
 
-    const { span, updatedOptions } = createSpan("DigitalTwinsClient-decomissionModel", options);
+    const { span, updatedOptions } = createSpan("DigitalTwinsClient-decommissionModel", options);
```

Comment:
Does this line cause a breaking change?

Reply:
I would not consider this a breaking change, especially as our tracing functionality is currently in preview.